src/LoginPage.js

```diff
@@ -0,0 +1,46 @@
+import React, { useState } from "react";
+import { Container, FormGroup, Label, Input, Button } from "reactstrap";
+
+function LoginPage() {
+
+  const [email, setEmail] = useState("");
+  const [password, setPassword] = useState("");
+
+  const forms = [
+    { text: "Email", name: "email", callback: setEmail },
```

Comment:
{ text: "Email", name: "email", setEmail } is more clear

Reply:
Done.